cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMapping.java

```diff
@@ -58,7 +58,7 @@ public RegulatingControlMappingForStaticVarCompensators forStaticVarCompensators
         return regulatingControlMappingForStaticVarCompensators;
     }
 
-    static class RegulatingControl {
+    public static class RegulatingControl {
```

Comment:
Just for me, the modifications in this class regards the updating of the original SSH export? If yes, it may be easier to review in another PR...

Reply:
Yes the modification is related to SSH export. The change was very small (only visibility of inner class), so we decided to include it here.